mqt/bench/tests/test_all.py

```diff
@@ -7,21 +7,19 @@
 from qiskit import QuantumCircuit
 
 from mqt.bench.benchmark_generator import get_one_benchmark
-from mqt.bench.benchmarks import (
+from mqt.bench.benchmarks import (  # hhl,; shor,
```

Comment:
Any particular reason for completely excluding these two instances here? 
Wouldn't it make sense to include at least some minimal tests for those circuits?

Reply:
Good point. I added `hhl` and `shor` now for test on the `indep` level and removed it from the other due to the long execution times.